Algorithms/Sorters/Comparison/CombSorter.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Algorithms.Sorters.Comparison
+{
+    /// <summary>
+    /// Comb sort is a relatively simple sorting algorithm that improves on bubble sort.
+    /// </summary>
+    /// <typeparam name="T">Type of array element.</typeparam>
+    public class CombSorter<T> : IComparisonSorter<T>
+    {
+        /// <summary>
+        /// Sorts array using specified comparer,
+        /// internal, in-place, unstable,
+        /// time complexity: O(n^2),
+        /// space complexity: O(1),
+        /// where n - array length.
```

Comment:
Could you expand this section a bit to include this info?
![image](https://user-images.githubusercontent.com/29201949/95549693-67133a80-0a10-11eb-81ba-e30e40ff06c0.png)

Reply:
And a link to the Wikipedia article

Reply:
Done